src/scripts/main.js

```diff
@@ -1,7 +1,19 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const elem = document.createElement('div');
```

Comment:
```suggestion
  const notification = document.createElement('div');
```
check variable names